internal/website/gatherexamples/gatherexamples.go

```diff
@@ -238,8 +254,8 @@ rewrite:
 			if err == nil {
 				blankImports = append(blankImports, path)
 			}
-		case strings.HasPrefix(lineContent, signifierCommentPrefix):
-			// "This example is used in" comment. Skip it.
+		case strings.Contains(lineContent, signifierComment):
+			// Signifier comment. Skip it.
```

Comment:
"signifier" doesn't mean much to me on its own. Perhaps we should refer to it as example inclusion comment or inclusion comment?

Reply:
Renamed to `inclusionComment`.